tests/gathering/test_indices_selector.py

```diff
@@ -0,0 +1,28 @@
+import os
+import json
+
+from urbansearch.gathering import indices_selector
+import config
+
+
+def test_relevant_indices_from_gz_file():
+    ind_sel = indices_selector.IndicesSelector(cities=['Delft', 'De Bilt'])
+    _file = os.path.join(config.get('resources', 'test'), 'domain-nl-0000.gz')
+    relevant = ind_sel.relevant_indices_from_file(_file)
+    assert len(relevant) == 0
```

Comment:
This is a test to see if it works for no co-occurences, right? 
Could you also write a test to check if len(relevant) == 1 for a gz file containing a correct co-occurence?